src/singletons/helper/LoggingChannel.cpp

```diff
@@ -109,7 +109,7 @@ void LoggingChannel::addMessage(MessagePtr message)
 
 QString LoggingChannel::generateOpeningString(const QDateTime &now) const
 {
-    QString ret = QLatin1Literal("# Start logging at ");
+    QString ret("# Start logging at ");
```

Comment:
I think the direct migration would be `QString ret = QStringLiteral("...");` but it doesn't really matter

Reply:
yea QLatin1String() would work, QString has QLatin1String constructor and picked it because it looks cleaner